leetcode/data-structure-1/contains-duplicate.py

```diff
@@ -57,7 +66,14 @@ def containsDuplicate3(self, nums: List[int]) -> bool:
         # if exists return True
         # if not, put the number in to dict
         # finally if nothing found return False
-        pass
+        vistited_numbers = {}
+        for num in nums:
+            if num in vistited_numbers.keys():
+                return True
+            else:
+                dict[num] = "None"
```

Comment:
Please check this line. dict variable should be renamed. Also the None I meant was not string, its the null equivalent of Python.